CONTRIBUTING.md

```diff
@@ -39,8 +39,8 @@
 
 ## Running the test application
 
-- Run `yarn prepare` after any changes to the toolkit to build it to be consumed by the documentation app.
-- Run `yarn start:documentation` to start the documentations apps server
+- Run `yarn watch:toolkit` - This will automatically detect changes in the toolkit and build it to be consumed by the documentation app.
+- In a separate terminal, run `yarn start:documentation` to start the documentations apps server
```

Comment:
might be a good idea to mention that this is using `fb-watchman` and maybe link to their docs.